integration_tests/examples/aac_rtp_input.rs

```diff
@@ -130,8 +120,8 @@ fn start_example_client_code() -> Result<()> {
     info!("[example] Start pipeline");
     examples::post("start", &json!({}))?;
 
-    stream_video(IP, INPUT_1_PORT, bunny_path.clone())?;
-    stream_audio(IP, INPUT_2_PORT, bunny_path, "aac")?;
+    start_ffmpeg_send_video(IP, INPUT_1_PORT, bunny_path.clone())?;
+    start_ffmpeg_send_audio(IP, INPUT_2_PORT, bunny_path, "aac")?;
```

Comment:
shouldn't this be start_ffmpeg_send_sample?

consider making all the functions that are not meant to be used directly private and only expose that we want to use in tests